_posts/2017-01-31-git_v_PhpStormu.md

```diff
@@ -0,0 +1,57 @@
+
+Oproti tomu [Github](http://github.com/) je internetová služba, která git využívá a díky ní je spolupráce ve více lidech velmi zjednodušená. V základu je zdarma a v článku ho budu používat spolu s gitem. 
+
+#Proč zrovna git?
+Git byl stvořený hlavně pro práci s textovými soubory - programátorské kódy jsou přeci jen snůška texťáků, proto git funguje hlavně pro kodéry a jejich práci. Dobře bude fungovat třeba i pro psaní knihy, blogu (tento blog je taky verzovaný v gitu) nebo i bakalářky či diplomky. 
+ 
+Git poskytuje pohled na texty v čase. Díky tomu se už nemusí ukládat víc verzí jednoho dokumentu ručně. Navíc velmi zjednodušuje spolupráci více lidí najednou nebo udržování více paralelních verzí - tam se projeví jeho pravá síla. 
+
+Git je primárně určen pro ovládání z příkazové řádky. Většina IDE ho ovšem má integrovaný, proto je možné používat klávesové zkratky a rozbalovaní meníček přímo v editoru. A to bude máš případ - zbytek článku budu popisovat práci v PhpStormu. Jestli máte zájem se naučit používat git nativně, dva odkazy na kvalitní návody jsem připojil na konec článku a až na tomto blogu přibude článek, dozvíte se to tady. 
+
+#Praxe - jdeme na to!
+Jestli se chceš s gitem naučit v příkazovém řádku (což je těžší, ale užitečnější), přeskoč na [další článek, který vyjde týden po tomto](http://jakpsatphp.cz/git-v-prikazovem-radku). Zbytek článku je psán pro PhpStorm 2016, práce v novější verzi ale bude velmi podobná. 
+
+> Všechny praktické rady budou formátovány tímto způsobem - pokud chceš rychle načíst jen teorii, tyto pasáže přeskakuj - ovšem nedoporučuju to, protože půlka mojí práce potom přijde nazmar. 
+
+##Repozitář
+Místo, kde se všechna gitovská magie odehrává, se nazývá **repozitář** nebo i **repo**. Repozitář je složka v počítači, kterou git sleduje. Dá se určit, že se nějaké soubory sledovat nemají (například lokální konfigurace), nemůžeme ale do repozitáře zahrnout soubory mimo tuto složku. 
+
+> Založ si nový gitový repozitář, tedy postupně: vytvoř si nový projekt v PhpStorm a vyber z horní nabídky _VCS_ (zkratka z version control systems) > _Enable Version Controll Integration..._, tam vyber _git_ a potvrď. Tím se vytvořil gitový repozitář pro celý projekt. Když se teď podíváš do složky s projektem, objevila se tam nová složka _.git_ Pozor, je označená systémem jako skrytá, takže bude viditelná až po povolení zobrazování skrytých souborů. Jestli ji nevidíš, nemusíš to moc řešit, není to příliš důležité. Když tuto složku smažeš, efektivně tím zrušíš sledování repozitáře, kde složka _.git_ byla. 
+
+##Commit
+Základním kamenem pro práci s gitem je **commit**. Commit je zachycení stavu souborů tak, jak zrovna jsou. Commit jako takový není dál dělitelný, je tou nejmenší jednotkou, s kterou se v gitu potkáš. Commity se řadí za sebou v čase, jeden následuje druhý tak, jak byly postupně pořizovány. 
```

Comment:
`Commit jako takový není dál dělitelný, je tou nejmenší jednotkou, s kterou se v gitu potkáš` -> toto tvrzení bych upravil například na to, že je to nejmenší jednotka kterou uživatel potká při běžné práci